metricbeat/module/system/_meta/kibana/default/dashboard/Metricbeat-system-overview.json

```diff
@@ -67,13 +67,13 @@
           "searchSourceJSON": "{\"index\":\"metricbeat-*\",\"query\":{\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true,\"default_field\":\"*\"}},\"language\":\"lucene\"},\"filter\":[]}"
         },
         "title": "Hosts histogram by CPU usage [Metricbeat System]",
-        "uiStateJSON": "{\"vis\":{\"defaultColors\":{\"0% - 5%\":\"rgb(247,252,245)\",\"5% - 10%\":\"rgb(199,233,192)\",\"10% - 15%\":\"rgb(116,196,118)\",\"15% - 20%\":\"rgb(35,139,69)\"}}}",
+        "uiStateJSON": "{\"vis\":{\"defaultColors\":{\"0% - 20%\":\"rgb(247,252,245)\",\"20% - 40%\":\"rgb(199,233,192)\",\"40% - 60%\":\"rgb(116,196,118)\",\"60% - 80%\":\"rgb(35,139,69)\"}}}",
```

Comment:
Did you change these on purpose?

Reply:
Hmm, I didn't change those, they might be dynamic? I'm going to revert the changes just to be sure.